server/models/Profile.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require("mongoose");
+const timeSlot = new mongoose.Schema({ start: Date, end: Date })
```

Comment:
Not sure if this is the best approach here. We'll likely keep availability pretty basic just monday to friday between X and Y time. Date might not work the best in this case but we'll discuss this more when we work on the feature.

Reply:
Would be nice to ensure the end time is not larger than the start time too but not a huge deal for me right now. As mentioned before, when we dive into availability we'll spend more time on this.

Reply:
Alright, gonna make this as independent column for better usage in future